Components/Queues/Drivers/StompQueueDriver.php

```diff
@@ -93,13 +101,34 @@ public function getStompVersion()
     }
 
     /**
-     * @param string $stompVersion
+     * Set the version to Stomp driver
+     *
+     * @param string $version
      */
-    public function setStompVersion($stompVersion)
+    public function setStompVersion(string $version = self::STOMP_VERSION)
     {
-        $this->stompVersion = $stompVersion;
+        $this->stompVersion = $version;
+    }
+
+    /**
+     * Set the timeout to Stomp driver
+     *
+     * @param int $timeout
+     */
+    public function setTimeout(int $timeout = 3)
```

Comment:
The timeout default value should be defined in a constant

Reply:
This parameter on the past was set on configure functions with a value 3 by default. Because of this, after refactor the configure function I set 3 in this function. In any case, we can set as null by default if you think is better.

Reply:
In which part it was defined as 3 in the past?

Reply:
This was defined in the configure method. This is the old method signature:
public function configure($host, $username, $password, $format = QueueDriverInterface::FORMAT_JSON, $version = self::STOMP_VERSION, $vhost = null, $timeout = 3, $sync = true)

Reply:
:+1: 

Exaclty, and it was hardcoded to 3 in the `configure()` method: 
https://github.com/smartboxgroup/integration-framework-bundle/commit/bdc51ee9d0e3c216389e02fb8090ecf53292f564#diff-363ac7ef637f2b5f991b745646f09dc8L155